core/src/i18n/pl.yaml

```
@@ -117,7 +119,65 @@ discovery-and-network:
 functions:
   title: Funkcje
   variable:
-    tooltip: Ta zmienna pochodzi z zasobu. Aby uzyskać wartość, kliknij link.
+    buttons:
+      add: Dodaj zmienną środowiskową
+      open:
+        custom: Zmienna niestandardowa
+        secret: Zmienna z sekretu
+        config-map: Zmienna z mapy konfiguracji
+    errors:
+      empty: Nazwa zmiennej jest pusta.
+      duplicated: Zmienna o takiej nazwie ju istnieje.
+      invalid: "Nazwa zmiennej musi składać się ze znaków alfanumerycznych, może zawierać również '_', bez spacji"
+      invalid-config: Select the config map name and key
+      invalid-secret: Select the secret name and key
+      restricted: Ta nazwa zmiennej jest zastrzerzona i nie może by użyta. Spróbuj użyć innej nazwy.
+    form:
+      config-map: $t(functions.variable.type.config-map)
+      key: $t(functions.variable.header.key)
+      name: Nazwa
+      secret: $t(functions.variable.type.secret)
+      value: $t(functions.variable.header.value)
+    header:
+      key: Klucz
+      name: Nazwa zmiennej
+      source: Żródło
+      value: Wartość
+    not-found: Ta funkcja nie posiada jeszcze zmiennych środowiskowych
+    not-match: Nie znaleziono zmiennych środowiskowych pasujących do wyszukiwanej frazy.
+    title:
+      create-modal:
+        custom: Stwórz Zmienną niestandardową
+        secret: Stwórz Secret Variable
+        config-map: Stwórz Config Map Variable
+      edit-modal:
+        custom: Edytuj Zmienną niestandardową
```

```suggestion
        custom: Edytuj zmienną niestandardową
```
It has been decided that we will use lowercase names in polish (if the name can be translated). Please check https://kubernetes.io/pl/docs/contribute/localization-pl/
Please apply it also in other places.